sample-remote-monitor-plugin/src/test/java/org/opensearch/alerting/SampleRemoteMonitorPluginIT.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting;
+
+import org.junit.Assert;
+import org.opensearch.test.OpenSearchIntegTestCase;
+
+public class SampleRemoteMonitorPluginIT extends OpenSearchIntegTestCase {
```

Comment:
What's this file for?

Reply:
When I was troubleshooting a bit last week, I recall seeing that the build would fail when using `excludeTestsMatching` and no test files matching the `**/*IT.class` pattern. I suspect that's why he included this dummy test; so there would always be a test that matches.